webpack/private/webpack_bundle.bzl

```diff
@@ -221,7 +221,10 @@ def _impl(ctx):
     if ctx.attr.chdir:
         env["JS_BINARY__CHDIR"] = ctx.attr.chdir
     for (key, value) in ctx.attr.env.items():
-        env[key] = " ".join([expand_variables(ctx, exp, attribute_name = "env") for exp in expand_locations(ctx, value, [ctx.attr.entry_point] + ctx.attr.srcs + ctx.attr.deps + ctx.attr.data).split(" ")])
+        env[key] = " ".join([
+            expand_variables(ctx, exp, attribute_name = "env")
+            for exp in expand_locations(ctx, value, [ctx.attr.entry_point] if ctx.attr.entry_point else ctx.attr.entry_points.keys() + ctx.attr.srcs + ctx.attr.deps + ctx.attr.data).split(" ")
```

Comment:
add some brackets so the order of operations is clear?
```
([ctx.attr.entry_point] if ctx.attr.entry_point else ctx.attr.entry_points.keys()) + ctx.attr.srcs + ctx.attr.deps + ctx.attr.data
```

Reply:
actually; probably better to assign to a variable at the top of the for look to make it readable

Reply:
Updated